sdk/src/main/java/ly/count/android/sdk/ModuleFeedback.java

```diff
@@ -470,11 +485,15 @@ void reportFeedbackWidgetManuallyInternal(@Nullable CountlyFeedbackWidget widget
 
             if (widgetInfo.type == FeedbackWidgetType.nps) {
                 if (!"nps".equals(typeInData)) {
-                    L.w("[ModuleFeedback] type in widget info does not match the type in widget data");
+                    L.w("[ModuleFeedback] type in widget info does not match the type in widget data nps");
```

Comment:
mention the recorded type:
"[ModuleFeedback] type in widget info [#givenValue] does not match the expected value for this type [nps]"